build-test.sh

```diff
@@ -52,16 +55,21 @@ initTargetDistroRid()
     if [ "$__PortableBuild" == 1 ]; then
         if [ "$__BuildOS" == "Linux" ]; then
             export __DistroRid="linux-$__BuildArch"
+            export __RuntimeId="linux-$__BuildArch"
         elif [ "$__BuildOS" == "OSX" ]; then
             export __DistroRid="osx-$__BuildArch"
+            export __RuntimeId="osx-$__BuildArch"
+        elif [ "$__BuildOS" == "FreeBSD" ]; then
+            export __DistroRid="freebsd-$__BuildArch"
+            export __RuntimeId="freebsd-$__BuildArch"
```

Comment:
Do we already have packages being published with this RID? I suspect this could cause some problems in package restore, or once we start using this in official builds.